.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,15 @@
-> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.
```

Comment:
GitHub doesn't render (didn't at least) properly linted files :(

Reply:
looks fine to me...

Reply:
how do you check it? could you send a link please?

Reply:
https://github.com/iterative/dvc.org/blob/lint-more/.github/PULL_REQUEST_TEMPLATE.md looks identical to https://github.com/iterative/dvc.org/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
that's not a fair link though :) we should be trying it on a PR description/comment. Create a PR on top of this one? Or put that MD into a comment.

Reply:
> You may disregard these recommendations if you used the **Edit on GitHub**
> button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the
[Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs)
list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description
of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to
[allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

Reply:
fixed